prepare.php

```diff
@@ -38,12 +38,44 @@
 // Replace variables in the wp-config.php file
 log_message( 'Replacing variables in wp-tests-config.php' );
 $contents = file_get_contents( $WPT_PREPARE_DIR . '/wp-tests-config-sample.php' );
+// Log system information to same directory as test run log
+$system_logger = <<<EOT
+// Create the log directory to store test results
+if ( ! is_dir(  __DIR__ . '/tests/phpunit/build/logs/' ) ) {
+	mkdir( __DIR__ . '/tests/phpunit/build/logs/', 0777, true );
+}
+// Log environment details that are useful to have reported.
+\$env = array(
+	'php_version'    => phpversion(),
+	'php_modules'    => array(),
+	'system_utils'   => array(),
+);
+\$php_modules = array(
+	'imagick',
+	'filter',
+	'xml',
+	'pcre',
+	'mod_xml',
+	'bcmath',
+);
+foreach( \$php_modules as \$php_module ) {
+	\$env['php_modules'][ \$php_module ] = phpversion( \$php_module );
+}
+\$ret = shell_exec( 'convert --version' );
+preg_match( '#Version: ImageMagick ([^\s]+)#', \$ret, \$matches );
+\$env['system_utils']['imagemagick'] = isset( \$matches[1] ) ? \$matches[1] : false;
+\$env['system_utils']['ghostscript'] = shell_exec( 'gs --version' );
+file_put_contents( __DIR__ . '/tests/phpunit/build/logs/env.json', json_encode( \$env, JSON_PRETTY_PRINT ) );
+EOT;
+$logger_replace_string = '// wordpress/wp-config.php will be ignored.' . PHP_EOL;
```

Comment:
Only if the file was _saved_ on Windows. Is this a use-case we need to solve for?